pkg/yqlib/doc/usage/hcl.md

```diff
@@ -7,7 +7,7 @@ HCL is commonly used in HashiCorp tools like Terraform for configuration files.
 - String interpolation and expressions (preserved without quotes)
 - Comments (leading, head, and line comments)
 - Nested structures (maps and lists)
-- Syntax colorization when enabled
+- Syntax colorisation when enabled
```

Comment:
You may configure misspell to use UK English via your .golangci.yaml file

https://golangci-lint.run/docs/linters/configuration/#misspell